Lesson_3/calculator/calc.js

```diff
@@ -0,0 +1,39 @@
+document.addEventListener('DOMContentLoaded', () => {
+   const whatMathOperation = prompt('Яку з доступних дій (add +, sub -, mult *, div /) бажаєте виконати?');
+
+   if (whatMathOperation !== '+' && whatMathOperation !== 'add'
+   && whatMathOperation !== '-' && whatMathOperation !== 'sub'
+   && whatMathOperation !== '*' && whatMathOperation !== 'mult'
+   && whatMathOperation !== '/' && whatMathOperation !== 'div') {
+      return alert('Спробуйте вибрати одну з запропонованих дій:\n add\n +\n sub\n -\n mult\n *\n div\n /');
```

Comment:
it will be better to move to the else block after all `if's`, to avoid double-check

Reply:
Looks like you have this check in the `else` block

Reply:
this check is here because I want to stop the script, if the user enters an incorrect value before asking for `number1` and `number2`

i delete `else`  as redundant

Reply:
I described the logic and the meaning in the prev lesson) It's redundant to check all possible variants in a case when you will check it separately in the nest steps)

Reply:
moved this check to the else block